src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -118,12 +120,27 @@ private void OnProjectChanged(IProjectSubscriptionUpdate e)
             _isDisabled = disableFastUpToDateCheckString != null && string.Equals(disableFastUpToDateCheckString, TrueValue, StringComparison.OrdinalIgnoreCase);
 
             _msBuildProjectFullPath = e.CurrentState.GetPropertyOrDefault(ConfigurationGeneral.SchemaName, ConfigurationGeneral.MSBuildProjectFullPathProperty, _msBuildProjectFullPath);
-            foreach (var referenceSchema in ReferenceSchemas)
+            _markerFile = e.CurrentState.GetPropertyOrDefault(ConfigurationGeneral.SchemaName, ConfigurationGeneral.CopyUpToDateMarkerProperty, null);
```

Comment:
In https://github.com/Microsoft/msbuild/pull/2213 I'd like to change this to be an item instead of a property. Reasoning is embedded there. Is that a problem?